web/src/components/tasks/business-formation/review/ReviewPartnership.tsx

```diff
@@ -12,9 +12,10 @@ export const ReviewPartnership = (): ReactElement => {
   const notEnteredText = (question: string): ReactElement => {
     return (
       <>
-        <div>
-          <em>{Config.formation.general.notEntered}</em> - {question}
-        </div>
+        <span className={"bg-accent-warm-extra-light text-italic"}>
+          {Config.formation.general.notEntered}
```

Comment:
can we use the <NotEntered> component here?

Reply:
The styling doesn't work because the `Not Entered` portion has to be a part of the first sentence but if we use the component then there are two sections, the left and right and it doesn't look like one sentence flowing into the second line anymore.